addons/mail/models/discuss/discuss_channel.py

```diff
@@ -228,7 +228,7 @@ def create(self, vals_list):
                 if cmd[0] != 0:
                     raise ValidationError(_('Invalid value when creating a channel with memberships, only 0 is allowed.'))
                 for field_name in cmd[2]:
-                    if field_name not in ["partner_id", "guest_id", "unpin_dt", "last_interest_dt", "fold_state"]:
+                    if field_name not in ["is_owner", "partner_id", "guest_id", "unpin_dt", "last_interest_dt", "fold_state"]:
```

Comment:
This is not acceptable, as it would mean anybody could create a new member which is owner, defeating the whole purpose.